.changeset/smooth-guests-join.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/backend-common': patch
+'@backstage/backend-test-utils': patch
+'@backstage/plugin-catalog-backend': patch
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+More prep work for better-sqlite3
```

Comment:
Perhaps modify the existing changeset instead?

Reply:
good point - moved over!